internal/webhooks/handler.go

```diff
@@ -29,32 +29,46 @@ const (
 	envOTELExporterOTLPEndpoint = "OTEL_EXPORTER_OTLP_ENDPOINT"
 	envOTELTracesExporter       = "OTEL_TRACES_EXPORTER"
 	envOTELResourceAttrs        = "OTEL_RESOURCE_ATTRIBUTES"
-
-	exporterOTLP   = "otlp"
-	exporterJaeger = "jaeger-thrift-splunk"
-
-	annotationConfInjectionEnabled = "o11y.splunk.com/inject-config"
-	annotationStatus               = "o11y.splunk.com/injection-status"
-	annotationReason               = "o11y.splunk.com/injection-reason"
+	envJavaToolsOptions         = "JAVA_TOOL_OPTIONS"
+
+	volumeName        = "splunk-instrumentation"
+	initContainerName = "splunk-instrumentation"
+	javaJVMArgument   = " -javaagent:/splunk/splunk-otel-javaagent-all.jar"
+	exporterOTLP      = "otlp"
+	exporterJaeger    = "jaeger-thrift-splunk"
+
+	annotationJava   = "o11y.splunk.com/inject-java"
+	annotationConfig = "o11y.splunk.com/inject-config"
+	annotationStatus = "o11y.splunk.com/status"
+	annotationReason = "o11y.splunk.com/reason"
```

Comment:
I see `o11y.splunk.com/injection-<status|reason>` is renamed to `o11y.splunk.com/<status|reason>` but this PR doesn't rely on it. What is the reasoning for that? Do we want to reuse this attribute for reporting any other statuses/reasons?

Also looks like we report only `"error"` status. Shouldn't we report successfully injected events?

And using "o11y.splunk.com/reason" attribute for reporting error messages doesn't seem apparent to me.

Maybe we can use?
`o11y.splunk.com/inject-java/status` always reported if `o11y.splunk.com/inject-java=true`
`o11y.splunk.com/inject-java/error` added only if `o11y.splunk.com/inject-java/status=error`

Reply:
Renamed it to fit with the other annotation but we renamed that to `inject-*` so renaming these back to `injection-status/reason`. Also added `success` annotation when there are no errors. We can elaborate this later and also emit k8s events in case of success/.failure.